addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2363,8 +2363,11 @@ const Wysiwyg = Widget.extend({
     },
     _onSelectionChange() {
         if (this.odooEditor.autohideToolbar) {
+            const selectionInLink = document.querySelector('span[data-o-link-zws="end"]');
             const isVisible = this.linkPopover && this.linkPopover.el.offsetParent;
-            if (isVisible && !this.odooEditor.document.getSelection().isCollapsed) {
+            const closestLinkEL = selectionInLink && selectionInLink.closest('a');
```

Comment:
I think the selectionInLink has become obsolete, and closestLinkEl is probably enought on it's own.

Could you please try to refactor your change to be as minimal as possible and only keep the `closestLinkEL`.

Sometink like 
```javascript
if (isVisible && ( !closestLinkEL || targetEL !== closestLinkEL)) {
....
```
should suffice I think.

Reply:
@sebgeelen Done changes :+1: